backend/views/emails/org_invite.html

```diff
@@ -7,7 +7,10 @@
 </p>
 
 <p>
-Please click <a href="<%= config.domain %>/#/activate/<%= company.realm %>/<%= token %>">here</a> to activate your account. 
+Please click <a href="<%= config.indabaDomain %>/activate/<%= company.realm %>/<%= token %>">here</a> to activate your account.
+</p>
+<p>
+However, if you already have an account with Amida, you can simply login <a href="<%= config.indabaDomain %>/login/">here</a>.
```

Comment:
Do we know if this ever happens? Would the user have just been readded if they already had an account?

Reply:
Honestly, I don't know. I was just being thorough, but we can remove it.

Reply:
Removing.